geo/src/main/java/com/github/davidmoten/geo/LatLong.java

```diff
@@ -61,4 +63,21 @@ public String toString() {
         return builder.toString();
     }
 
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        final LatLong latLong = (LatLong) o;
+        return Double.compare(lat, latLong.lat) == 0 && Double.compare(lon, latLong.lon) == 0;
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(lat, lon);
```

Comment:
perf: will implicitly create two boxed Double's and an array under the hood - so would be better to do something like:
```
    return 31 * Double.hashCode(lat) +
            Double.hashCode(lon);
```